src/backend/common/MemoryManagerImpl.hpp

```diff
@@ -253,7 +253,7 @@ void MemoryManager<T>::printInfo(const char *msg, const int device) {
     printf("%s\n", msg);
     printf(
         "---------------------------------------------------------\n"
-        "|     POINTER      |    SIZE    |  AF LOCK  | USER LOCK |\n"
+        "|  DEVICE POINTER  |    SIZE    |  AF LOCK  | USER LOCK |\n"
```

Comment:
This is not strictly true for a memory manager that handles pinned memory.

Reply:
@umar456: But it's also the case that the pointer in question is not, say, the `af_array` - which I thought for a while it might be. If not here, I wonder if there's some other place to clarify what pointer this message refers to?

Reply:
Perhaps, something like `DEVICE/PINNED-MEMORY POINTER` I know it is long but it caters to both the situations.

If not this, we can add a note to after horizontal line in line 296. something like the following.
```c
printf("---------------------------------------------------------\n"); // This is already there at end of table
printf("POINTER here is either device or pinned memory pointer\n");
printf("---------------------------------------------------------\n"); // new horizontal line
```
something along those lines.

I like the second approach personally.

Reply:
@9prady9 I think that sort of information would be in the documentation rather than printed every time.

I understand your concern but this would also be misleading.

Reply:
Some times the proximity of this info helps a round trip to the docs or code, given that it is just few words.

Reply:
I don't think the table itself is actually documented anywhere and, as a new user, I did not know how to interpret any of the headers.
I'll amend the pull request to document `AF LOCK  | USER LOCK` somewhere and can document pointer at the same time.

Is `DEVICE/PINNED-MEM POINTER` or `DEVICE/PINNED-MEM PTR` still legible while carrying more information?

Reply:
How about DATA or BUFFER POINTER instead of DEVICE POINTER?

Reply:
Maybe? It's very clear to me what "device pointer" means because it's used all the time in CUDA.

Reply:
I've added some documentation explaining the table (see c90fa9f).

Reply:
Although you have added the table header description in the function documentation, you haven't changed the first column header name to just 'POINTER'

Reply:
Fixed.